olm-catalog/serverless-operator/manifests/serverless-operator.clusterserviceversion.yaml

```diff
@@ -41,29 +41,29 @@ spec:
   apiservicedefinitions: {}
   customresourcedefinitions:
     owned:
-    - description: Represents an installation of a particular version of Knative Serving
-      displayName: Knative Serving
-      kind: KnativeServing
-      name: knativeservings.operator.knative.dev
-      statusDescriptors:
-      - description: The version of Knative Serving installed
-        displayName: Version
-        path: version
-      - description: Conditions of Knative Serving installed
-        displayName: Conditions
-        path: conditions
-        x-descriptors:
-        - 'urn:alm:descriptor:io.kubernetes.conditions'
-      version: v1alpha1
-    - description: Represents an installation of a particular version of Knative Eventing
-      displayName: Knative Eventing
-      kind: KnativeEventing
-      name: knativeeventings.operator.knative.dev
-      statusDescriptors:
-      - description: The version of Knative Eventing installed
-        displayName: Version
-        path: version
-      version: v1alpha1
+      - description: Represents an installation of a particular version of Knative Serving
```

Comment:
This diff is quite odd. When i run `make csv` I don't get the same diff :thinking:. Are we having different versions of `yq` or something?